modules/web/src/app/cluster/details/cluster/rbac/add-service-account-dialog/template.html

```diff
@@ -0,0 +1,60 @@
+<!--
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<div class="dialog">
+  <km-dialog-title>Create Service Account</km-dialog-title>
```

Comment:
I think dialog title should match the "Add Service Account" button which opens this dialog. Wdyt?
Same for the "Create Service Account" button in this dialog.